tests/test_schemas.py

```diff
@@ -99,6 +99,10 @@ def test_anonymous_request(self):
                         url='/example/custom_list_action/',
                         action='get'
                     ),
+                    'custom_list_action': coreapi.Link(
```

Comment:
Note that it wouldn't be valid to have a dictionary with the same key repeated twice.